server/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const db = require('../database-mongodb/index.js');
+const Product = require('../database-mongodb/Product.js');
+
+const app = express();
+const PORT = 3002;
+
+
+app.use(express.static(__dirname + '/../client/dist'));
```

Comment:
Here's a suggestion that eslint had me change after I configured it to use the Airbnb style guide. You'll also need to require the `path` module at the top of your file.

```suggestion
app.use(express.static(path.join(__dirname, '/../client/dist')));
```